server/service/src/sync/test/integration/central_server_configurations.rs

```diff
@@ -5,17 +5,27 @@ use reqwest::{Client, Url};
 use serde_json::json;
 use util::{hash::sha256, uuid::uuid};
 
-use crate::sync::{settings::SyncSettings, SyncApiV5, SyncConnectionError, SyncCredentials};
+use crate::sync::{
+    api::{SyncApiError, SyncApiV5},
+    settings::SyncSettings,
+    SyncCredentials,
+};
 
 impl SyncApiV5 {
     pub(crate) async fn upsert_central_records(
         &self,
         value: &serde_json::Value,
-    ) -> Result<(), SyncConnectionError> {
-        self.create_post("/sync/v5/test/upsert", value)?
-            .send()
-            .await?
-            .error_for_status()?;
+    ) -> Result<(), SyncApiError> {
+        self.do_post("/sync/v5/test/upsert", value).await?;
+
+        Ok(())
+    }
+
+    pub(crate) async fn delete_central_records(
+        &self,
+        value: &serde_json::Value,
+    ) -> Result<(), SyncApiError> {
+        self.do_post("/sync/v5/test/delete", value).await?;
```

Comment:
why not include them into the main file? because they are test only? For the other apis I used name reflecting the actual http call, e.g. `post_test_upsert` or `post_test_delete` and then use more descriptive names/comments where the api is used. No change request think it doesn't matter for tests...

Reply:
Yeah this is only used in tests, didn't think we ned to expose them in SyncApiV5 struct. I think we will have an ability to edit central records via similar API when we start doing central configurations for things like [master lists](https://github.com/openmsupply/open-msupply/issues/269) and programs, at which point it's probably good to change this tests to use the new methods in SyncApiV5 struct.